frappe/contacts/doctype/contact/contact.py

```diff
@@ -22,49 +22,72 @@ class Contact(Document):
 		from frappe.core.doctype.dynamic_link.dynamic_link import DynamicLink
 		from frappe.types import DF
 
+		_updated_at: DF.Datetime | None
 		address: DF.Link | None
+		can_inbox: DF.Check
 		company_name: DF.Data | None
 		department: DF.Data | None
 		designation: DF.Data | None
 		email_id: DF.Data | None
 		email_ids: DF.Table[ContactEmail]
+		first_message_time: DF.Datetime | None
 		first_name: DF.Data | None
+		form_id: DF.Data | None
+		form_name: DF.Data | None
 		full_name: DF.Data | None
 		gender: DF.Link | None
 		google_contacts: DF.Link | None
 		google_contacts_id: DF.Data | None
 		image: DF.AttachImage | None
+		inserted_at: DF.Datetime | None
 		is_primary_contact: DF.Check
+		last_incoming_call_time: DF.Datetime | None
+		last_message_time: DF.Datetime | None
 		last_name: DF.Data | None
+		last_outgoing_call_time: DF.Datetime | None
+		lead_owner: DF.Link | None
 		links: DF.Table[DynamicLink]
 		middle_name: DF.Data | None
 		mobile_no: DF.Data | None
+		page_url: DF.Data | None
+		pancake_conversation_id: DF.Data | None
+		pancake_customer_id: DF.Data | None
+		pancake_page_id: DF.Data | None
+		pancake_updated_at: DF.Datetime | None
 		phone: DF.Data | None
 		phone_nos: DF.Table[ContactPhone]
+		phone_number_provided_time: DF.Datetime | None
+		psid: DF.Data | None
 		pulled_from_google_contacts: DF.Check
+		remote_ip: DF.Data | None
 		salutation: DF.Link | None
+		source: DF.Link | None
+		source_group: DF.Literal["Facebook", "Zalo", "Tiktok", "Phone", "Website Form", "Email", "Orther"]
```

Comment:
I said `Other` not `Orther`
```suggestion
		source_group: DF.Literal["Facebook", "Zalo", "Tiktok", "Phone", "Website Form", "Email", "Other"]
```